gravitee-am-ui/src/app/domain/settings/users/user/profile/profile.component.html

```diff
@@ -21,46 +21,69 @@
       <form (ngSubmit)="update()" fxLayout="column" #userForm="ngForm">
         <div class="gv-form-section">
           <div class="gv-form-section-title">
-            <h5>Username</h5>
-            <small>The username of the user.</small>
+            <h3>{{ isServiceAccount() ? 'Service Name' : 'Username' }}</h3>
```

Comment:
it can be extracted to method

Reply:
I dont think it make sens to create metod only for this use.